samples/qa-a11y/src/views/PopupMenu.js

```diff
@@ -0,0 +1,86 @@
+import BodyText from '@enact/agate/BodyText';
+import Button from '@enact/agate/Button';
+import PopupMenu from '@enact/agate/PopupMenu';
+import React from 'react';
+
+import Section from '../components/Section';
+import useBoolArray from '../components/useBoolArray';
+
+const PopupMenuView = () => {
+	const [open, handleOpen] = useBoolArray(5);
+
+	return (
+		<>
+			<Section title="Default">
+				<Button alt="Normal" onClick={handleOpen(0, true)}>Open 0</Button>
+				<Button alt="With a Title" onClick={handleOpen(1, true)}>Open 1</Button>
```

Comment:
I think it's out of scope but the title's color is black so it's not quite distinguishable.

Reply:
I wrote it in the PR description.